src/domains/sign/io_sign_issuer_func.tf

```diff
@@ -78,12 +76,11 @@ module "io_sign_issuer_func" {
 
 module "io_sign_issuer_func_staging_slot" {
   count  = var.io_sign_issuer_func.sku_tier == "PremiumV3" ? 1 : 0
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app_slot?ref=v4.1.3"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app_slot?ref=v6.0.1"
```

Comment:
why are you using `v6.0.1` here and `v6.2.1` in `module.io_sign_issuer_func`? can't version be the same for all modules?

Reply:
Because `v6.0.1` is the latest version in which the `function_app_slot` module has been updated